src/Chapter02/Listing02.13.UsingTheNewlineCharacterToInsertANewline.cs

```diff
@@ -1,14 +1,14 @@
 namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter02.Listing02_13
 {
-    #region INCLUDE
     public class DuelOfWits
     {
         public static void Main()
         {
+            #region INCLUDE
             System.Console.Write(
                 "\"Truly, you have a dizzying intellect.\"");
             System.Console.Write("\n\"Wait 'til I get going!\"\n");
```

Comment:
Do we want to use `Environment.NewLine` here or is it still too early in the book? Or just use Console.WriteLine maybe?
```suggestion
            System.Console.WriteLine("\"Wait 'til I get going!\"");
```

Reply:
The text is explicit showing the difference between `WriteLine` and `Write`.